frontend/angular-js/README.md

```diff
@@ -1,17 +1,26 @@
 # AngularJS Tech Guides [Tech Guides under construction]
 
-## Kickoff Guide
+## Content
 
-At [Wolox](http://wolox.com.ar), we have developed the [Angular kickoff guide](./docs/kickoff/README.md) to create our Angular projects. Check it out to find a detailed guide to start with your project.
+- [Project Bootstrapper](#project-bootstrapper)
+- [Style Guide](#style-guide)
+- [Open Source](#open-source)
+- [Useful Documentation](#useful-documentation)
```

Comment:
I'd delete the content of this section for now. just leave the title.

Reply:
add internal and external libraries like in react but with empty content for now.